internal/config/config.go

```diff
@@ -231,3 +259,22 @@ func newCloudConnectorsConfig() CloudConnectorsConfig {
 		ResourceID:    os.Getenv(CloudResourceIDEnvVar),
 	}
 }
+
+const (
+	CloudbeatGCPListAssetPageSizeEnvVar = "CLOUDBEAT_GCP_LIST_ASSETS_PAGE_SIZE"
+	CloudbeatGCPListAssetTimeoutEnvVar  = "CLOUDBEAT_GCP_LIST_ASSETS_TIMEOUT"
+)
+
+func overwritesFromEnvVars(c *Config) {
```

Comment:
I suggest to log errors that can occur while overwriting

Reply:
Done